spring-integration-core/src/main/java/org/springframework/integration/annotation/IntegrationComponentScan.java

```diff
@@ -22,15 +22,23 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Import;
+import org.springframework.core.annotation.AliasFor;
 import org.springframework.integration.config.IntegrationComponentScanRegistrar;
 
 /**
- * Configures component scanning directives for use with @{@link org.springframework.context.annotation.Configuration} classes.
- * Scan Spring Integration specific components.
+ * Configures component scanning directives for use with
+ * the {@link org.springframework.context.annotation.Configuration} classes.
+ * <p>
+ * Scans Spring Integration specific components, like {@link MessagingGateway}
```

Comment:
I don't think we should be ambiguous here `s/like//` - it currently __only__ scans for `MessagingGateway`.

`Scans for {@link MessagingGateway} on interfaces to create {@code GatewayProxyFactoryBean}s.`